common/common-db/src/main/java/org/ow2/proactive/db/TransactionHelper.java

```diff
@@ -94,6 +94,8 @@ public TransactionHelper(SessionFactory sessionFactory) {
         for (int i = 0; i <= maximumNumberOfRetries; i++) {
             try {
                 return tryExecuteTransaction(sessionWork, readWriteTransaction, readOnlyEntities);
+            } catch (DatabaseManagerException e) {
```

Comment:
I think there are IllegalArgumentException thrown as well, can you check if there are other expected exception ?